docs/release-notes.md

```diff
@@ -7,17 +7,67 @@ weight: 150
 We recommend always using "tagged" versions of Ondat rather than "latest",
 and to perform upgrades only after reading the release notes.
 
-The latest tagged release is `2.6.0`. For
+The latest tagged release is `2.7.0`. For
 installation instructions see our
 [Install](/docs/reference/cluster-operator/install) page.
 
-The latest CLI release is `2.6.0`, available from
+The latest CLI release is `2.7.0`, available from
 [GitHub](https://github.com/storageos/go-cli/releases).
 
 # Upgrading
 
 To upgrade from version 1.x to 2.x, contact Ondat [support](/docs/support) for assistance.
 
+## 2.7.0 - Released 2022-04-06
+
+### New
+
+k8s & Orchestrator Rolling Upgrade
+
+- Kubernetes rolling upgrade for AWS EKS, Google Anthos, Google GKE, Microsoft Azure, Openshift and Rancher. Other platforms will be supported in future releases
+
+Operator
+
+- Skipped duplicated nodes by hostname during sync
+- Updated memory limit
+- Introduced topology spread constraint with `ScheduledAnyway`
+
+API Manager
+
+- Adds a feature so when a PVC is not found scheduling will not be blocked
+
+Control Plane
+
+- Set `Only_Numeric_Owners` to true on NFSv4 setting on Ganesha
+
+Data Plane
+
+- Removed support for FUSE. Ondat now only supports TCMU. `target_core_user` must now be used. Read [System Configuration](https://docs.ondat.io/docs/prerequisites/systemconfiguration/) for more information
+- Rewrote the RPC interface between the Control Plane and the Data Plane. All of the old `ctl` tools have been removed and we have added a single `supctl` tool that can be used to debug/program the DP from the command-line
+- Removed the 32-bit mappings and uses the UUIDs passed by the CP directly to address presentations and deployments.
+  > ⚠️ If you decide to upgrade to 2.7.0 and want to downgrade, you can only roll back to 2.6.0, not earlier versions. Roll back instruction can be found [here](/docs/operations/...)
+
+### Fixed
+
+Operator
+
+- Fixed a bug that sometimes caused the operator to enter a deadlock state after Ondat cluster CR object deletion
+- Fixed a bug where the Ondat Operator v2.7.0 does not work with OpenShift 4.7
```

Comment:
I don't think this should be here - it's not a change as the issue didn't exist in any released version of the product.